packages/babel-compat-data/.npmignore

```diff
@@ -1,4 +1,3 @@
+build
 scripts
-src
-test
```

Comment:
Why remove src and test?

Reply:
There is no src or test directory in this module. But I can also leave it in. Should I?

Reply:
Oh I edited my comment exactly at the same time as your reply :sweat_smile: